portal/templates/_partials/beta-banner.njk

```diff
@@ -0,0 +1,13 @@
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+
+<div class="govuk-width-container">
+  {{ govukPhaseBanner({
+    tag: {
+      text: "beta"
+    },
+    attributes: {
+      "data-cy": "beta-banner"
+    },
+    html: 'This is a new service – your <a class="govuk-link" data-cy="beta-feedback-link" href="/feedback">feedback</a> will help us to improve it.'
```

Comment:
Please add target='_blank' so current user page is not disturbed

Reply:
done